docs/csharp/language-reference/keywords/return.md

```diff
@@ -10,22 +10,25 @@ helpviewer_keywords:
 ms.assetid: 6da6e152-5b58-4448-8f3f-470dd0617ecd
 ---
 # return (C# Reference)
-The `return` statement terminates execution of the method in which it appears and returns control to the calling method. It can also return an optional value. If the method is a `void` type, the `return` statement can be omitted.  
-  
- If the return statement is inside a `try` block, the `finally` block, if one exists, will be executed before control returns to the calling method.  
-  
-## Example  
- In the following example, the method `CalculateArea()` returns the local variable `area` as a [double](../../../csharp/language-reference/keywords/double.md) value.  
-  
- [!code-csharp[csrefKeywordsJump#6](../../../csharp/language-reference/keywords/codesnippet/CSharp/return_1.cs)]  
-  
-## C# Language Specification  
- [!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]  
-  
+
+The `return` statement terminates execution of the method in which it appears and returns control to the calling method. It can also return an optional value. If the method is a `void` type, the `return` statement can be omitted.
+
+ If the return statement is inside a `try` block, the `finally` block, if one exists, will be executed before control returns to the calling method.
+
+## Example
+
+ In the following example, the method `CalculateArea()` returns the local variable `area` as a [double](double.md) value.
+
+ [!code-csharp[csrefKeywordsJump#6](codesnippet/CSharp/return_1.cs)]
```

Comment:
The file does not exist (this PR removes it), and there's no corresponding file currently in the dotnet/samples repo.